scripts/examples/build-all-clusters-app.py

```diff
@@ -54,6 +59,12 @@ def main():
 
   e = IDFExecutor()
 
+  if args.generate_flash_script:
+    if os.getenv('ESPPORT') is None:
```

Comment:
It gets written into the defaults for the flash script.

Reply:
@sweetymhaiske  - if we leave the default blank, will the flash script respect the ESPPORT environment variable?

Reply:
nevermind - taking out the warning. If it's not set, you'll get a weird error, but I suppose that's what the readme is for. Removing environment variables from the build environment is out of scope.

Reply:
> @sweetymhaiske - if we leave the default blank, will the flash script respect the ESPPORT environment variable?

In that case, it will try to find out any serial port available.